deploy/helm/templates/events-collector-deployment.yaml

```diff
@@ -83,6 +83,10 @@ spec:
                   name: {{ template "common.secret" . }}
                   key: SOLARWINDS_API_TOKEN
                   optional: true
+            - name: GOMEMLIMIT
+              valueFrom:
+                resourceFieldRef:
+                  resource: limits.memory
```

Comment:
This uses the configured memory limits of the Container. If they are not configured, Kubernetes fills this value with Node allocatable.